guiEditor/src/stringTools.ts

```diff
@@ -0,0 +1,87 @@
+export class StringTools {

```

Comment:
Don't we have some sort of file saver in the framework already? I am pretty sure I have seen similar code. Doesn't NME have the same thing?